Source/DataSources/ModelVisualizer.js

```diff
@@ -235,9 +269,7 @@ define([
     }
 
     function onModelReady(model) {
-        model.activeAnimations.addAll({
```

Comment:
This removes turning on all model animations when the model is loaded. Still need to find a way to manually turn on animations when loading models through the entity framework so the Sandcastle models example works as it did before